src/components/FormCard.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { Card } from '@material-ui/core';
+
+const StyledCard = styled(Card)`
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    padding: 2rem;
+  }
+  padding: 0.5rem;
+`;
+
+interface UserCardProps {
+  children: ReactNode;
+}
+
+const FormCard: React.FC<UserCardProps> = ({ children }) => <StyledCard variant="outlined">{children}</StyledCard>;
```

Comment:
kunne evt satt variant i props slik at vi kan bruke denne til flere stiler

Reply:
går det an å velge en default variant for alle komponenter i themes ? vi bruker vel hovedsaklig en type ?

Reply:
tja, vet ikke. vi trenger ikke å bry oss om dette nå (designet vil jo bli endret uansett)